docs/documentation/pages/DECKHOUSE_CONFIGURE.md

```diff
@@ -78,40 +102,53 @@ data:
     - 10.2.200.55
 ```
 
+Some modules can also be configured using custom resources. Use the search bar at the top of the page or select a module in the left menu to see a detailed description of its settings and the custom resources used.
+
 ### Enabling and disabling the module
 
 > Depending on the [bundle used](#module-bundles), some modules may be enabled by default.
 
-To enable/disable a module, add the `<moduleName>Enabled` parameter to the `deckhouse` ConfigMap with one of the following two values: `"true"` or `"false"` (note: quotation marks are mandatory), where `<moduleName>` is the name of the module in camelCase.
+To enable/disable the module, set `spec.enabled` field of the `ModuleConfig` custom resource to `true` or `false`. It may require creating `ModuleConfig` resource for the module.
```

Comment:
```suggestion
To enable/disable the module, set `spec.enabled` field of the `ModuleConfig` custom resource to `true` or `false`. Note that this may require you to first create a `ModuleConfig` resource for the module.
```